File: js files/3c.js
// 3c: Write a function in JS that accepts a string and returns the number of vowels in the string

function countVowels(text) {
    const vowels = "aeiaoAEIAO";
    let count = 0; 

    for (let char of text) {
        if (vowels.includes(char)) {
            count++;
        }
    }

    return count;
}

const userInput = prompt("Enter a string:");

const vowelCount = countVowels(userInput);

console.log(result);

File: py files/3c.py
# 3c: Write a function in Python that accepts a string and returns the number of vowels in the string

def count_vowels(text):
    """Returns the number of vowels in a string"""
    vowels = "aeiouAEIOU"
    count = 0
    
    for char in text:
        if char in vowels:
            count += 1
    
    return count

if __name__ == "__main__":
    user_input = input("Enter a string: ")
    vowel_count = count_vowels(user_input)
    print(f"The string '{user_input}' contains {vowel_count} vowels")